engine-rest/engine-rest-openapi/src/main/templates/paths/history/cleanup/jobs/get.ftl

```diff
@@ -0,0 +1,67 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/history-cleanup/get-history-cleanup-jobs/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "findCleanupJobs"
+      tag = "History Cleanup"
+      summary = "Find clean up history jobs (GET)"
+      desc = "Finds history cleanup jobs (See
+              [History cleanup](${docsUrl}/user-guide/process-engine/history/#job-progress))."
```

Comment:
The link does not exist anymore since 7.10, as far as I can see.

Reply:
Same as above. Updated to this one: https://docs.camunda.org/manual/7.9/user-guide/process-engine/history/#history-cleanup